src/components/SlateEditor/plugins/definitionList/handlers/onBackspace.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Editor, Element, Transforms, Path, Node, Range, Point } from 'slate';
+import hasNodeOfType from '../../../utils/hasNodeOfType';
+import { TYPE_DEFINTION_DESCRIPTION, TYPE_DEFINTION_LIST, TYPE_DEFINTION_TERM } from '../types';
+
+const onBackspace = (
+  e: KeyboardEvent,
+  editor: Editor,
+  nextOnKeyDown: ((e: KeyboardEvent) => void) | undefined,
+) => {
+  if (!editor.selection) return nextOnKeyDown && nextOnKeyDown(e);
+  const isDefinition = hasNodeOfType(editor, TYPE_DEFINTION_LIST);
+
+  if (!isDefinition) {
+    return nextOnKeyDown && nextOnKeyDown(e);
+  }
+
+  const [selectedDefinitionItem, selectedDefinitionItemPath] = Editor.parent(
+    editor,
+    editor.selection.anchor.path,
+  );
+
+  if (
+    Element.isElement(selectedDefinitionItem) &&
+    selectedDefinitionItem.type === TYPE_DEFINTION_TERM
+  ) {
+    const [selectedTerm, selectedTermPath] = [selectedDefinitionItem, selectedDefinitionItemPath];
+
+    const [, firstItemNodePath] = Editor.node(editor, [...selectedTermPath, 0]);
+    // If cursor is placed at start of first item child
+    if (
+      Point.equals(Range.start(editor.selection), Editor.start(editor, firstItemNodePath)) &&
+      !Path.hasPrevious(selectedTermPath)
+    ) {
+      e.preventDefault();
+      Editor.withoutNormalizing(editor, () => {
+        Transforms.removeNodes(editor, { at: Path.next(selectedTermPath) });
+        Transforms.liftNodes(editor, { at: selectedTermPath });
+      });
+      return;
+    } else if (Range.start(editor.selection) && Node.string(selectedTerm) === '') {
+      e.preventDefault();
+      Transforms.select(editor, Editor.end(editor, Path.previous(selectedTermPath)));
+      Editor.withoutNormalizing(editor, () => {
+        Transforms.removeNodes(editor, { at: Path.next(selectedTermPath) });
+        Transforms.removeNodes(editor, { at: selectedTermPath });
+      });
+
+      return;
+    }
+  } else if (
+    Element.isElement(selectedDefinitionItem) &&
+    selectedDefinitionItem.type === TYPE_DEFINTION_DESCRIPTION
+  ) {
+    const [selectedDescription, selectedDescriptionPath] = [
+      selectedDefinitionItem,
+      selectedDefinitionItemPath,
+    ];
```

Comment:
Hva er formålet med dette?

Reply:
For å gjøre det lettere å lese flyten da du vil alltid jobbe i `par` med `DT` og `DD`.  Kunne gått kjørt med kun `definitionlistItem` men ble veldig rotete når du jobbet gjennom koden. Fastsette de ulike elementene med DT/DD gjorde det lettere å lese hva som skjer.